src/main/scala/NumberNotations.scala

```diff
@@ -0,0 +1,69 @@
+
+object NumberNotations {
+
+  val englandShort = List("sextillion",
+    "quintillion",
+    "quadrillion",
+    "trillion",
+    "billion",
+    "million",
+    "thousand")
+
+  val european = List("trilliard",
+    "trillion",
+    "billiard",
+    "billion",
+    "milliard",
+    "million",
+    "thousand")
+
+  def notationSplit(num: String): List[String] = {
+    num.reverse.grouped(3).toList.map(_.reverse.toList.mkString("")).reverse
```

Comment:
mkString("") could be replaced with just mkString because "" is the default value